tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/api/rest/v1/entities/ProcessResponse.java

```diff
@@ -112,51 +158,10 @@ public boolean equals(Object o) {
     return Objects.equals(id, that.id)
         && Objects.equals(name, that.name)
         && Objects.equals(bpmnProcessId, that.bpmnProcessId)
-        && Arrays.equals(sortValues, that.sortValues)
+        && Objects.deepEquals(sortValues, that.sortValues)
         && Objects.equals(version, that.version)
         && Objects.equals(startEventFormId, that.startEventFormId)
-        && Objects.equals(tenantId, that.tenantId);
-  }
-
-  @Override
-  public int hashCode() {
-    int result = Objects.hash(id, name, bpmnProcessId, version, startEventFormId, tenantId);
-    result = 31 * result + Arrays.hashCode(sortValues);
-    return result;
-  }
-
-  @Override
-  public String toString() {
```

Comment:
[Question] Is there a reason for delete the toString method?

Reply:
@nathansandi I removed by mistake, I put it back :)